### DISCUSSION TITLE
Unaware skill

### DISCUSSION THREAD

**Discussion by JesterWizard** [MokhaLeee/fe8u-cskillsys-kernel#307] [General]
So I'm taking inspiration from Pokemon again to make the Unaware skill, which ignores stat changes of the enemy. 

It works, though it does require me to load the the battle target and actor in the weapon bonus and skill functions of every stat boost file. 

My question is, how do I make it also appy to the weaken status? As that's the only thing stopping me right now.

![image](https://github.com/user-attachments/assets/c4b1641f-edc3-424d-89e7-f2f7d6777ec9)

**Comment by MokhaLeee** [comment]
I think they should belong to pre-battle calc

**Reply by JesterWizard** [reply]
Been a while, but on reflection yes this is probably best.
